### ISSUE TITLE
Not working on Zabbix 5.0

### ISSUE THREAD

**Issue by Ludo-code** [liuk001/zabbix-email-html-templates#1]
To work on Zabbix 5.0 we must cut emoji

**Comment by liuk001** [comment]
I think it's a known limit of emoji support in notification for oldest zabbix version. Emoji support started in Zabbix 6.0 if I'm not wrong.

**Comment by Ludo-code** [comment]
Yes it's for this i have do a pr #2 with a second branch containing the same code without emoji

**Comment by Ludo-code** [comment]
but thinking i have do a little error in the pr because request to merge in main
